hw/ip/otbn/dv/otbnsim/sim/wsr.py

```diff
@@ -149,17 +149,90 @@ def set_unsigned(self, value: int) -> None:
         self._random_value = value
 
 
+class URNDWSR(WSR):
+    '''Models URND PRNG Structure'''
+    def __init__(self, name: str):
+        super().__init__(name)
+        self.traced = False
+        self._seed = [0x84ddfadaf7e1134d, 0x70aa1c59de6197ff,
+                      0x25a4fe335d095f1e, 0x2cba89acbe4a07e9]
+        self.state_256b = [self._seed,
+                           4 * [0], 4 * [0],
+                           4 * [0], 4 * [0]]
+        self.URND = 4 * [0]
+        self._next_value = 0
+        self.start_prng = False
```

Comment:
A better name is probably something like `running`. "start" sounds like it might be just be true as we start, but this becomes true and stays true.